jumanji/environments/routing/connector/env.py

```diff
@@ -180,31 +173,26 @@ def step(
             grid=grid, step_count=state.step_count + 1, agents=agents, key=state.key
         )
 
-        # Construct timestep: get observations, rewards, discounts
-        grids = self._obs_from_grid(grid)
+        # Construct timestep: get rewards, discounts
```

Comment:
Discounts are not computed here anymore, is this comment still relevant ?

Reply:
Done is computed, will change the docstring to dones instead of discount